ipapython/ipautil.py

```diff
@@ -245,6 +245,21 @@ def set_ip_net(self, ifnet):
         self._net = ifnet
 
 
+class CheckedIPAddressLoopback(CheckedIPAddress):
+    """IPv4 or IPv6 address with additional constraints with
+    possibility to use a loopback IP.
+    Reserved or link-local addresses are never accepted.
+    """
+    def __init__(self, addr, parse_netmask=True, allow_multicast=False):
+
+        super(CheckedIPAddressLoopback, self).__init__(
+                addr, parse_netmask=parse_netmask, allow_loopback=True,
```

Comment:
Personally, if I were to change one specific option here, I would probably add it as a last argument, but that's up to you, I guess.